src/components/App.tsx

```diff
@@ -39,6 +41,32 @@ class App extends React.Component<Record<string, unknown>, AppState> {
     // TODO: fetch real ETH price
     const ethPriceUsd = 2700;
     this.fetchAave(this.mainAccount, ethPriceUsd);
+
+    // TODO: confirm which provider -- alternative below
+    // const httpProvider = this.web3.eth.providers["HttpProvider"];
+
+    const httpProvider = new Web3.providers.HttpProvider(
+      "https://mainnet.infura.io/v3/a305eb1a2c9b42cdbcf61db762f8243e"
+    );
+
+    console.log("current web3 provider: ", httpProvider);
+
+    let txBuilder;
+    let lendingPool;
+
+    txBuilder = new TxBuilderV2(Network.mainnet, httpProvider);
+    console.log("txbuilder: ", txBuilder);
+
+    lendingPool = txBuilder.getLendingPool(Market.Proto); // get all lending pool methods
+    console.log("lending pool: ", lendingPool);
+
+    let lendingResponse = await lendingPool.deposit({
+      user: "0xC33D36523FBF8360792C2c372f5fE457BeeC001f",
+      reserve: "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+      amount: "10000",
+    });
```

Comment:
per the docs, this should return the methods to create aave lending pool transactions. This is what the response looks like:
![image](https://user-images.githubusercontent.com/6494625/116821009-c95a9680-ab2c-11eb-9c90-0daa97cd1ee3.png)
I think from here, we use the web3 library to trigger transactions for the user to sign, such as the ERC20 approval and ultimately lending. I'll try to emit these transactions next

cc @adotcruz

Reply:
added logic to actually trigger the transactions